packages/compass-aggregations/src/modules/pipeline-builder/pipeline-ai.ts

```diff
@@ -121,6 +133,11 @@ function logFailed(errorMessage: string) {
       errorMessage,
     }
   );
+  track('AI Response Failed', () => ({
+    editor_view_type,
```

Comment:
Why do we need this? Is it something that can be just derived from a user journey in amplitude?

Reply:
A user could do both at once which might potentially mess up the journey. Mostly following the tracking plan here.

Reply:
How can you run into this for CRUD and two excluding view types of Agg Builder? 🤔

Reply:
Could it happen if a user has multiple tabs or windows open and runs an ai request in each? Maybe it's a non-issue and we can get rid of it.

Reply:
As far as I see in the code inflight requests are getting cancelled on unmount and tabs are getting re-mounted, so doesn't look like it will happen